### PR TITLE
media-video/webcamoid: version bump to 8.7.1

### PR DESCRIPTION
**Pull request by PPed72**
Closes: https://bugs.gentoo.org/729042
Signed-off-by: Paolo Pedroni <paolo.pedroni@iol.it>
Package-Manager: Portage-3.0.9, Repoman-3.0.2

### CODE REVIEW

File: media-video/webcamoid/webcamoid-8.7.1.ebuild [hunk 1]
```diff
@@ -0,0 +1,97 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PLOCALES="ca de el es et fr gl he it ja kab ko nb_NO nl pl pt pt_BR ru tr uk zh_CN zh_TW"
+inherit l10n qmake-utils
```
Comment (juippis) [inline media-video/webcamoid/webcamoid-8.7.1.ebuild:7]: Just install all locales unconditionally, helps you to clean this ebuild a bit too.

Comment (PPed72) [reply media-video/webcamoid/webcamoid-8.7.1.ebuild]: Done.

**Review by juippis — COMMENTED** [review]
```
 * Package:    media-video/webcamoid-8.7.1
 * Repository: testworld
 * Maintainer: reagentoo@gmail.com proxy-maint@gentoo.org
 * USE:        abi_x86_64 alsa amd64 elibc_glibc ffmpeg gstreamer headers kernel_linux libuvc userland_GNU
 * FEATURES:   network-sandbox preserve-libs sandbox userpriv usersandbox
>>> Unpacking source...
>>> Unpacking webcamoid-8.7.1.tar.gz to /var/tmp/portage/gpm/portage/media-video/webcamoid-8.7.1/work
>>> Source unpacked in /var/tmp/portage/gpm/portage/media-video/webcamoid-8.7.1/work
>>> Preparing source in /var/tmp/portage/gpm/portage/media-video/webcamoid-8.7.1/work/webcamoid-8.7.1 ...
>>> Source prepared.
>>> Configuring source in /var/tmp/portage/gpm/portage/media-video/webcamoid-8.7.1/work/webcamoid-8.7.1 ...
 * Running qmake ...
Info: creating stash file /var/tmp/portage/gpm/portage/media-video/webcamoid-8.7.1/work/webcamoid-8.7.1/.qmake.stash
 [ ok ]
>>> Source configured.
>>> Compiling source in /var/tmp/portage/gpm/portage/media-video/webcamoid-8.7.1/work/webcamoid-8.7.1 ...
make -j16 -l10 
cd libAvKys/ && ( test -e Makefile || /usr/lib64/qt5/bin/qmake -o Makefile /var/tmp/portage/gpm/portage/media-video/webcamoid-8.7.1/work/webcamoid-8.7.1/libAvKys/libAvKys.pro 'QMAKE_AR=x86_64-pc-linux-gnu-ar cqs' QMAKE_CC=x86_64-pc-linux-gnu-gcc QMAKE_LINK_C=x86_64-pc-linux-gnu-gcc QMAKE_LINK_C_SHLIB=x86_64-pc-linux-gnu-gcc QMAKE_CXX=x86_64-pc-linux-gnu-g++ QMAKE_LINK=x86_64-pc-linux-gnu-g++ QMAKE_LINK_SHLIB=x86_64-pc-linux-gnu-g++ QMAKE_OBJCOPY=x86_64-pc-linux-gnu-objcopy QMAKE_RANLIB= QMAKE_STRIP= 'QMAKE_CFLAGS=-march=native -O2 -pipe -frecord-gcc-switches' QMAKE_CFLAGS_RELEASE= QMAKE_CFLAGS_DEBUG= 'QMAKE_CXXFLAGS=-march=native -O2 -pipe -frecord-gcc-switches' QMAKE_CXXFLAGS_RELEASE= QMAKE_CXXFLAGS_DEBUG= 'QMAKE_LFLAGS=-Wl,-O1 -Wl,--as-needed -Wl,--defsym=__gentoo_check_ldflags__=0' QMAKE_LFLAGS_RELEASE= QMAKE_LFLAGS_DEBUG= CONFIG+=debug PREFIX=/usr BUILDDOCS=0 INSTALLDEVHEADERS=1 INSTALLQMLDIR=/usr/lib64/qt5/qml LIBDIR=/usr/lib64 NOAVFOUNDATION=1 NODSHOW=1 NOVCAMWIN=1 NOWASAPI=1 NOV4L2=1 NOCOREAUDIO=1 NOJACK=1 NOOSS=1 NOPULSEAUDIO=1 NOQTAUDIO=1 NOV4LUTILS=1 NOVIDEOEFFECTS=1 ) && make -f Makefile 
Project ERROR: QtQuick Controls 2 required.
make: *** [Makefile:48: sub-libAvKys-make_first-ordered] Error 3
 * ERROR: media-video/webcamoid-8.7.1::testworld failed (compile phase):
 *   emake failed
```
Well that was fast. Looks like you're missing a dep or something.

**Comment by PPed72** [comment]
Strange: I didn't change a thing in dependencies.

Do you have the same error with current in-tree version, BTW?

**Comment by juippis** [comment]
Yes, *media-video/webcamoid-8.1.0-r1* compiles and installs fine. I wonder if *qtquickcontrols* got forked to a new project that's now required by *webcamoid*, or if this is something that can be solved using cmake's `find_package`.

**Comment by PPed72** [comment]
It now needs qtquickcontrols2, I've checked. Fixed commit in a few minutes...

**Comment by juippis** [comment]
Oh we have that in tree already... I failed to see it after searching for it...

**Review by juippis — APPROVED** [review]

**Comment by juippis** [comment]
Few things I noticed: This doesn't actually install any locales now, and I get a message about pre-compressed man files. But those can be solved later.

**Pull request closed without merging**